src/include/storage/storage_structure/disk_array.h

```diff
@@ -104,9 +101,9 @@ struct PIPUpdates {
 class DiskArrayInternal {
 public:
     // Used when loading from file
-    DiskArrayInternal(FileHandle& fileHandle, DBFileID dbFileID, common::page_idx_t headerPageIdx,
-        BufferManager* bufferManager, WAL* wal, transaction::Transaction* transaction,
-        bool bypassWAL = false);
+    DiskArrayInternal(FileHandle& fileHandle, DBFileID dbFileID,
```

Comment:
Should this now all be `BMFileHandle`?

Reply:
Yes. It probably could have been changed in one of the previous PRs.